src/rest-server/src/controllers/v2/job.js

```diff
@@ -43,6 +44,8 @@ const update = asyncHandler(async (req, res) => {
   const jobName = res.locals.protocol.name;
   const userName = req.user.username;
   const frameworkName = `${userName}~${jobName}`;
+  const userToken = req.headers.authorization;
```

Comment:
what if the token expires when pod starts

Reply:
Then the init container will failed and return as platform error.
Currently, we don't have service account config so pass user token here. 
I think it's just a short-term solution, and current permission check avoid to create a generate token for runtime. We can fix this in future

---
In reply to: [348965232](https://github.com/microsoft/pai/pull/3875#discussion_r348965232) [](ancestors = 348965232)